src/python/pants/backend/python/goals/tailor_test.py

```diff
@@ -182,7 +182,7 @@ def test_find_putative_targets_subset(rule_runner: RuleRunner) -> None:
 
 
 def test_find_putative_targets_for_entry_points(rule_runner: RuleRunner) -> None:
```

Comment:
Is there some kind of end-to-end (or integration or smoke or whatever) test for `tailor`? I had all the tests passing and everything looking prim and proper and decided to try generating a `BUILD` file and saw that it wasn't what I expected.

Surely y'all would've spotted the bug, but that didn't give me great confidence in my tests :/

Instead it'd be nice if I could dump the files like this test does, but then actually run `tailor` and assert the output.